packages/manager/modules/pci/src/projects/project/additional-ips/service/service.js

```diff
@@ -4,20 +4,42 @@ export default class PciProjectAdditionalIpService {
     this.$http = $http;
   }
 
-  getRegions(projectId, ovhSubsidiary, planCode) {
+  getRegions(projectId, ovhSubsidiary, planCode, addonFamily) {
+    const params = { ovhSubsidiary };
+
+    if (planCode) {
+      params.planCode = planCode;
+    }
+
+    if (addonFamily) {
+      params.addonFamily = addonFamily;
+    }
```

Comment:
```suggestion
    const params = { ovhSubsidiary, planCode, addonFamily };
```
Could it be simplified like this ?